desktop-app/src/desktop-app.ts

```diff
@@ -14,10 +14,19 @@ initialize();
 let mainWindow;
 
 function createWindow () {
+  let newWidth = 1000;
+  let newHeight = 600;
+  try {
+    const mainScreen = screen.getPrimaryDisplay();
+    const dimensions = mainScreen.size;
+    newWidth = Math.max(newWidth, Math.round(dimensions.width * 0.7));
+    newHeight = Math.max(newHeight, Math.round(dimensions.height * 0.85));
+  } catch {}
```

Comment:
why is it in try/catch, what if it fails? no fallback values are provided?

Reply:
@agilob The fallback values are above the try, 1000 and 600. I don’t know if it can fail but I put it like that just in case some OS refuse to give the values.